kpaamcamcommands/analyze.py

```diff
@@ -0,0 +1,135 @@
+"""
+
+"""
+import pathlib
+import itertools
+
+import pandas
+from lingpy import LexStat, Alignments, csv2list
+from lingpy.convert.plot import plot_heatmap
+from lingpy.evaluate.acd import bcubes
+
+from lexibank_kpaamcam import Dataset
+
+
+def register(parser):
+    pass
+
+
+def run(args):
+    ds = Dataset()
+    cldf = ds.cldf_reader()
+
+    def outpath(fname):
+        return str(ds.dir / 'analyses' / fname)
```

Comment:
To make the command more robust - i.e. make sure it can be run from whatever `cwd` - we compute the output directory relative to the dataset's directory.